community/rfcs/24-05-16-OPEA-001-Overall-Design.md

```diff
@@ -58,7 +58,7 @@ The proposed overall architecture is
 
 The proposed OPEA workflow is
 
-![OPEA Workflow](opea_workflow.png "OPEA Workflow")
+![OPEA Workflow](opea_workflow.png)
```

Comment:
Actually the content in “” works fine without writing, as other locations can attest. But it will interfere with the tool's recognition of the reference. So I removed it.

Reply:
We have to merge this PR today, let us know if it's valuable to keep the title, we can add back in another PR.